runtime/Makefile

```diff
@@ -9,13 +9,11 @@ QIR_STDLIB_DIR?=$(MK_DIR)/qir-stdlib/target/release/
 QIR_STDLIB_INCLUDES_DIR?=$(MK_DIR)/qir-stdlib/target/release/build/include/
 CODE_COVERAGE?=OFF
 BUILD_TYPE?=Release
-ENABLE_KOKKOS?=OFF
-ifeq ($(ENABLE_KOKKOS),OFF)
-	LIGHTNING_GIT_TAG_VALUE?=latest_release
-else
-	# TODO: Update this to latest_release for the v0.30.0 release
-	LIGHTNING_GIT_TAG_VALUE?=main
-endif
+ENABLE_WARNINGS?=ON
+ENABLE_OPENMP?=ON
+ENABLE_LIGHTNING_KOKKOS?=OFF
+LIGHTNING_GIT_TAG_VALUE?=latest_release
+LIGHTNING_KOKKOS_GIT_TAG_VALUE?=v0.30.0
```

Comment:
I thought you mentioned kokkos would get a `latest_release` tag with the new release?

Reply:
It's not there yet :disappointed: Let's merge this PR without it.. I'll update it whenever this tag is created.

Reply:
Got it now worries 👍